web/app/views/environments/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="card">
+    <div class="card-block">
+	<h4 class="card-title">Environments for <%= @app.name =%></h4>
+	<div class="text-right mb-3">
+	    <%= link_to '+ New Environment', new_application_environment_path, class: "btn btn-primary" %>
+	</div>
+	<table class="table table-striped">
+	    <thead>
+		<tr>
+		    <th>Name</th>
+		    <th>Production</th>
+        <th>Action</th>
+		</tr>
+	    </thead>
+	    <tbody>
+		<% @environments.each do |env| %>
+		    <tr scope="row">
+			<td>
+			    <%= link_to env.name, application_environment_path(@app, env) %>
+			</td>
+			<td>
+        <%= env.productionFlag ? "True" : "" %>
+			</td>
+      <td>
+        <button type="button" class="btn btn-secondary btn-sm">Copy</button>
+        <button type="button" class="btn btn-secondary btn-sm">Delete</button>
+      </td>
```

Comment:
A lot of these are I think because I'm using spaces and the file that I copied from (the application views, basically) was written with tabs - should I standardize on tabs in erb files?